engine/src/main/java/org/terasology/logic/inventory/ItemComponent.java

```diff
@@ -80,6 +80,10 @@
      */
     public int baseDamage = 1;
 
+    public boolean pickableUp = true;
```

Comment:
Sure thing! But what would I call the new component? The only thing that comes to mind is "PickableComponent", but that doesn't sound right.